src/main/resources/assets/plethora/lang/nl_be.lang

```diff
@@ -0,0 +1,148 @@
+
+# Neural interface/connector
+item.plethora.neuralConnector.name=Neurale connector
+item.plethora.neuralConnector.desc=Wordt gebruikt om verbinding te maken met neurale koppelstukken
+
+item.plethora.neuralInterface.name=Neuraal koppelstuk
+item.plethora.neuralInterface.desc=Gaat in je helm slot. Maakt verbinding met de neurale connector.
+
+gui.plethora.neuralInterface.peripherals=Bekijk randapparatuur
+gui.plethora.neuralInterface.modules=Bekijk modules
+
+# Redstone integrator
+tile.plethora.redstone_integrator.name=Redstone-integrator
+tile.plethora.redstone_integrator.desc=Redstone-niveaus ophalen en instellen voor deze blok
+
+# Upgrade adjectives
+adjective.plethora.clock=Chronal
+adjective.plethora.daylight_detector=Lichtgevoelige
+adjective.plethora.note_block=Acoustic
+
+# Minecart
+entity.plethora.plethora:minecartComputer.name=Mijnkar computer
+
+# Creative tab
+itemGroup.plethora=Plethora
+
+# Core config
+gui.config.plethora.costsystem=Kostenstelsel
+gui.config.plethora.costsystem.initial=Beginlevel
+gui.config.plethora.costsystem.regen=Regenereer
+gui.config.plethora.costsystem.limit=Limiet
+gui.config.plethora.costsystem.allowNegative=Sta negatieve buffer toe
+gui.config.plethora.blacklist=Zwarte lijst
+gui.config.plethora.blacklist.blacklistMods=Mods op de zwarte lijst
+gui.config.plethora.blacklist.blacklistModules=Modules op de zwarte lijst
+gui.config.plethora.blacklist.blacklistModulesTurtle=Turtle-modules op de zwarte lijst
+gui.config.plethora.blacklist.blacklistModulesPocket=Pocket-modules op de zwarte lijst
+gui.config.plethora.blacklist.blacklistModulesMinecart=Mijnkar-modules op de zwarte lijst
+gui.config.plethora.blacklist.blacklistModulesTile=Tegel-modules op de zwarte lijst
+gui.config.plethora.blacklist.blacklistProviders=Providers op de zwarte lijst
+gui.config.plethora.blacklist.blacklistTileEntities=TileEntiteiten op de zwarte lijst
+gui.config.plethora.basecosts=Basismethodekosten
+gui.config.plethora.testing=Testen
+gui.config.plethora.testing.bytecodeVerify=Verifiëer bytecode
+gui.config.plethora.testing.debug=Foutopsporing
+gui.config.plethora.testing.likeDocs=Documentatie vereisen
+gui.config.plethora.testing.strict=Strikte validatie
+
+# Gameplay config
+gui.config.plethora.laser=Frickin' laserstraal
+gui.config.plethora.laser.minimumPotency=Minimale potentie
+gui.config.plethora.laser.maximumPotency=Maximale potentie
+gui.config.plethora.laser.cost=Kostprijs
+gui.config.plethora.laser.damage=Schade
+gui.config.plethora.laser.lifetime=Levensduur
+
+gui.config.plethora.kinetic=Kinetische uitbreiding
+gui.config.plethora.kinetic.launchMax=Maximum lanceringsfactor
+gui.config.plethora.kinetic.launchCost=Lanceringskost
+gui.config.plethora.kinetic.launchYScale=Lancering Y schaal
+gui.config.plethora.kinetic.launchElytraScale=Elytra-lanceringsschaal
+gui.config.plethora.kinetic.launchFallReset=Lancering valafstandsreset
+gui.config.plethora.kinetic.launchFloatReset=Launcerings zweefreset
```

Comment:
Should be "lancerings" instead of "launcerings"

Reply:
✔️